clients/gtest/CMakeLists.txt

```diff
@@ -105,7 +105,6 @@ endif( )
 set( hipblas_test_common
   ../common/utility.cpp
   ../common/cblas_interface.cpp
-  ../common/clients_common.cpp
```

Comment:
Was the work started to move bench/test_harness instantiations to an object library shared across both test and bench clients like I did in rocBLAS?    Maybe that should still be done?    Should make a ticket but fine to delete for now if _gtest.cpp instantiates what it needs.

Reply:
Don't think so, aside from 64-bit interface changes the last major change I see to clients_common.cpp was when I changed the test infrastructure to yaml-based testing in late 2023.